app-i18n/uchardet/uchardet-9999.ebuild

```diff
@@ -7,10 +7,7 @@ inherit cmake-utils git-r3
 
 DESCRIPTION="An encoding detector library"
 HOMEPAGE="https://www.freedesktop.org/wiki/Software/uchardet/"
-EGIT_REPO_URI=(
-	https://anongit.freedesktop.org/git/${PN}/${PN}.git
-	git://anongit.freedesktop.org/${PN}/${PN}
-)
+EGIT_REPO_URI="https://anongit.freedesktop.org/git/${PN}/${PN}.git"
```

Comment:
> HTTPS is blocked in some places, e.g. public cafes, public networks in universities, etc.

What? If someone is blocking HTTPS, he's effectively blocking half of the Internet (and the number of sites forcing HTTPS is growing fast). There's really no point in putting any effort in supporting such a system. Git is not really obligatory in Gentoo, and I see no reason to make it 'work' more than plain HTTPS downloads which don't.